lib/hanami/assets/helpers.rb

```diff
@@ -396,8 +421,12 @@ def image(source, options = {})
       #   <%= favicon %>
       #
       #   # <link href="https://assets.bookshelf.org/assets/favicon-28a6b886de2372ee3922fcaf3f78f2d8.ico" rel="shortcut icon" type="image/x-icon">
+      #
+      # @example Enable Push Promise/Early Hints
+      #
+      #   <%= favicon 'favicon.ico', push: true %>
       def favicon(source = DEFAULT_FAVICON, options = {})
-        options[:href]   = asset_path(source)
+        options[:href]   = asset_path(source, push: options.delete(:push) || false, as: :image)
```

Comment:
WDYT, we really need to use `options.delete(:push) || false` here? can we use just `nil` as a false value?

Reply:
From the reader perspective, if we see only boolean values, we are sure that no accidental `nil` was leaked in.

Reply:
good point, thanks! 👍